### PR TITLE
Graphical Overmap update

### PR DESCRIPTION
**Pull request by Larwck**
#### Summary
SUMMARY: Mods "Update to Graphical Overmap mod"

#### Purpose of change
I felt the readme and modinfo needed more information to clarify some things. This mod is certainly a bit of a fuss to get working. I also don't think the merge caught my last update, which fixed some bg color issues.

#### Describe the solution
Noted in modinfo the importance of reading the readme. Clarified how to remove 5x5 location map from the sidebar in the readme. To further simplify the install have moved the font.png to the fonts folder.

### CODE REVIEW

**Comment by tinukedaya** [comment]
Any chance you would make the map work with smaller font size than 16? I really love how your map looks, but would like it even more if I can set it a bit smaller.

**Comment by Larwck** [comment]
> Any chance you would make the map work with smaller font size than 16? I really love how your map looks, but would like it even more if I can set it a bit smaller.

Thanks! This would require me to create all new icons of a smaller size, which is certainly possible but not something I'm planning on doing soon. 16px is a comfort zone for me when making pixel art. If anyone wants to make their own tileset, they can use mine as a reference and as long as each relevant icon is in the same position with a bit of fiddling with the options and fonts.json we could get it working fairly quickly.

**Comment by ZhilkinSerg** [comment]
I suggest moving font to fonts folder to simplify installation.

> 
> 
> Any chance you would make the map work with smaller font size than 16? I really love how your map looks, but would like it even more if I can set it a bit smaller.

8x8 would look gross

image

**Comment by tinukedaya** [comment]
> 8x8 would look gross
> 
> image

I was thinking more like 12x12 than 8. I can see that 8 would be loosing too much of it's details and look really bad.

**Comment by Kilvoctu** [comment]
A little late to the party, but I just applied this mod to my savegame. The font (graphics) were much too small for my display, as my overmap is typically at fontsize 28. To reduce the need to squint, I modified the map_font_LARWICK.png.
Figure I provide the pngs as options for any other users. These are for 24 and 32 fontsize respectively. I tested them and they work with no graphical issues, though 24 looks slightly distorted as it's not pixel perfect (still looks fine to me in-game).

map_font_LARWICK_24.png:
map_font_LARWICK_24

map_font_LARWICK_32.png:
map_font_LARWICK_32

File: data/mods/Graphical_Overmap/modinfo.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
     "name": "Graphical Overmap",
     "authors": [ "Larwick" ],
     "maintainers": [ "Larwick" ],
-    "description": "Gives the overmap a graphical overhaul. Please refer to readme for installation.",
+    "description": "Gives the overmap a graphical overhaul. Please refer to readme for installation or you will get an error.",
```
Comment (ZhilkinSerg) [inline data/mods/Graphical_Overmap/modinfo.json]:
```suggestion
    "description": "Gives the overmap a graphical overhaul.  Please refer to readme for installation or you will get an error.",
```

**Comment by ZhilkinSerg** [comment]
I suggest renaming `fonts.json` to something like `fonts.json.txt` (and update instruction accordingly).

**Comment by kevingranade** [comment]
This pull request has been mentioned on **Cataclysm: Dark Days Ahead**. There might be relevant details there:

https://discourse.cataclysmdda.org/t/no-sidebar-in-overmap-screen/22394/3

**Pull request merged**